### PR TITLE
libhandy 1.4.0

### PR DESCRIPTION
**Pull request by tschoonj**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/libhandy.rb [hunk 1]
```diff
@@ -13,13 +13,8 @@ class Libhandy < Formula
...
   end
 
-  # NOTE: The glade catalog is disabled due to a bug that has been fixed but
-  # not landed in a stable libhandy version yet (as of September 2020).
-  # See https://gitlab.gnome.org/GNOME/libhandy/-/merge_requests/614
-  # When it lands, -Dglade_catalog=enabled should work (make sure to add
-  # glade to the dependencies)
-
   depends_on "gettext" => :build
+  depends_on "glade" => :build
```
Comment (carlocab) [inline Formula/libhandy.rb:17]: Looks like this needs to be a runtime dependency.

**Comment by tschoonj** [comment]
I enabled support for `glade`, but made it only a build-time dependency: `glade` is linked here to a plugin only, usable from within `glade` exclusively, so thought it would make sense not to make it a run-time dependency, given that most people using `libhandy` will never need the `glade` plugin.

But this triggers a linkage test error...

I think that if there is no clean way to circumvent this test error, it is best to get rid of `glade` support entirely in this formula.

Any thoughts?

**Comment by carlocab** [comment]
On Linux there's a way to ignore linkage failures with the `allowed_missing_libraries` method. We could maybe port that to macOS if enabling the plugin is worth the effort.

**Comment by tschoonj** [comment]
`glade` has been broken for years on macOS so I wouldn't bother with it at all.

**Comment by carlocab** [comment]
Not bothering with it sounds good.

**Comment by tschoonj** [comment]
Done!

**Review by carlocab — APPROVED** [review]

**Pull request closed without merging**